betfund_bet365/response/pre_match_odds.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, data):
         super(PreMatchOddsResponse, self).__init__(data)
 
     @property
-    def events(self) -> Union[List[FiResultBase], None]:
+    def results(self) -> Union[List[FiResultBase], None]:
```

Comment:
Why the change here?

Reply:
Quick Fix

In order to standardize downstream usage of this wrapper I am making response object top level structure common for all objects.

**For Example**
-All responses contain two common keys:

```python
{
    "success": 1,
    "results": []
}
```

I was originally calling two of them `"events"` for some reason -- bringing these all to `"results"` because downstream caller will use one helper to ingest results object.

see: [Previous Pull Request](https://github.com/betfund/betfund-bet365/pull/2)

Reply:
Beautiful, totally makes sense.